docs/portal.rst

```diff
@@ -0,0 +1,208 @@
+=====================
+The Management Portal
+=====================
+
+#. Prerequisites_
+
+#. `Ready, Set, ASAP! (It's quick I promise)`_
+
+#. `The TL;DR section`_
+
+    - `Creating and testing React components`_
+    - Build_
+    - Preview_
+    - `Add images`_
+    - `Add Asset folder`_
+    - `Run Flask daemon`_
+    - `The Flask daemon`_
+    - `I npm ejected`_
+    - `TailwindCSS`_
+    - `Custom CSS styles`_
+    - `Apply custom CSS styles`_
+    - Troubleshooting_
+
+Prerequisites
+-------------
+You must have Pattoo already installed and the database set up.
+Please see installation_ documentation for more info.
+
+.. _installation: https://github.com/PalisadoesFoundation/pattoo/blob/master/docs/installation.rst#basic-installation
+
+Ready, Set, ASAP! (It's quick I promise)
+----------------------------------------
+
+Ready
+^^^^^
+Go in the portal directory in ``pattoo/api/portal/react``.
+
+Install project dependencies:
+
+.. code-block:: bash
+
+    ./pattoo/api/portal/react$ npm install
```

Comment:
This is not acceptable. The portal installation must be included in the overall installation. Having an extra hidden step will not work. The user must expect complete functionality upon installation **once**.